lib/web_ui/test/window_test.dart

```diff
@@ -82,7 +82,7 @@ void testMain() {
     ), useSingle: false);
     expect(window.browserHistory, isA<MultiEntriesBrowserHistory>());
 
-    Future<void> check<T>(String method, Map<String, Object?> arguments) async {
+    Future<void> check<T>(String method, dynamic arguments) async {
```

Comment:
prefer `Object?`

Reply:
(using `Object?` forces you to cast before calling things on it, so we'll catch more issues in analysis)

Reply:
Yep, you're right, I like `Object?` better! I'll change it! Thanks!